tests/local.postman_environment.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
This environment should be loaded into Postman to be tested along with the collection